v21.1/build-a-csharp-app-with-cockroachdb.md

```diff
@@ -36,168 +38,83 @@ Install the latest version of the [Npgsql driver](https://www.nuget.org/packages
 $ dotnet add package Npgsql
 ~~~
 
-<section class="filter-content" markdown="1" data-scope="secure">
-
-## Step 3. Create the `maxroach` user and `bank` database
-
-{% include {{page.version.version}}/app/create-maxroach-user-and-bank-database.md %}
-
-## Step 4. Generate a certificate for the `maxroach` user
-
-Create a certificate and key for the `maxroach` user by running the following command.  The code samples will run as this user.
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ cockroach cert create-client maxroach --certs-dir=certs --ca-key=my-safe-directory/ca.key
-~~~
-
-## Step 5. Convert the key file for use by C# programs
-
-The private key generated for user `maxroach` by CockroachDB is [PEM encoded](https://tools.ietf.org/html/rfc1421).  To read the key in a C# application, you will need to convert it into PKCS#12 format.
-
-To convert the key to PKCS#12 format, run the following OpenSSL command on the `maxroach` user's key file in the directory where you stored your certificates:
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ openssl pkcs12 -inkey client.maxroach.key -password pass: -in client.maxroach.crt -export -out client.maxroach.pfx
-~~~
-
-As of December 2018, you need to provide a password for this to work on macOS. See <https://github.com/dotnet/corefx/issues/24225>.
+## Step 4. Create a database
 
-## Step 6. Run the C# code
+{% include {{page.version.version}}/app/create-a-database.md %}
 
-Now that you have created a database and set up encryption keys, in this section you will:
-
-- [Create a table and insert some rows](#basic-example)
-- [Execute a batch of statements as a transaction](#transaction-example-with-retry-logic)
-
-### Basic example
-
-Replace the contents of `cockroachdb-test-app/Program.cs` with the following code:
-
-{% include copy-clipboard.html %}
-~~~ csharp
-{% include {{ page.version.version }}/app/basic-sample.cs %}
-~~~
+## Step 5. Run the C# code
 
-Then, run the code to connect as the `maxroach` user.  This time, execute a batch of statements as an atomic transaction to transfer funds from one account to another, where all included statements are either committed or aborted:
+Now that you have set up your project and created a database, you will run a basic application that creates a table and inserts some rows.
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ dotnet run
-~~~
+### Get the code
 
-The output should be:
+Replace the contents of the `Program.cs` file that was automatically generated in your `cockroachdb-test-app` directory with the code below:
 
-~~~
-Initial balances:
-	account 1: 1000
-	account 2: 250
+{% include_cached copy-clipboard.html %}
+~~~ c#
+{% remote_include https://raw.githubusercontent.com/cockroachlabs/hello-world-csharp/main/basic.cs %}
```

Comment:
I think we should use different branches of `hello-world-csharp` for CC and local. I had been creating a `cockroachcloud` branch for CC, and the `main` branch for local, on some of the hello world repos.

That way the user doesn't need to replace or alter as many of the connection parameters.

We'd need to make another filtered div here with local/cockroachcloud, with the correct `remote_include` path pointing to main/cockroachcloud.